package.json

```diff
@@ -32,13 +32,16 @@
     "@11ty/eleventy": "^0.8.3",
     "@11ty/eleventy-plugin-rss": "^1.0.6",
     "@11ty/eleventy-plugin-syntaxhighlight": "^2.0.3",
+    "@babel/cli": "^7.6.4",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
     "@browser-logos/chrome": "^1.0.12",
     "@browser-logos/firefox": "^2.0.5",
     "@browser-logos/safari": "^1.0.4",
     "autoprefixer": "^9.6.1",
     "babel-minify": "^0.5.1",
```

Comment:
We probably don't need this as direct dependency anymore, as we don't use their CLI.

Reply:
Removing…

Reply:
Marking this back as unresolved - it's specifically `babel-minify` line that I commented on that shouldn't be necessary anymore :)